### PR TITLE
Remove KOV from OVA

### LINKED ISSUE #473: Remove kov from OVA and docs

### PR DESCRIPTION
**Pull request by andrewtchin**
Fixes #473

### CODE REVIEW

**Comment by anchal-agrawal** [comment]
There's some references remaining in `fileserver/html/index.html` and `ovatools/vic-ova-ui/main.go`.

**Comment by andrewtchin** [comment]
fixed the console ui, leaving the html for josh

File: installer/ovatools/vic-ova-ui/main.go [hunk 1]
```diff
@@ -84,9 +84,6 @@ func main() {
 		if port, ok := ovf.Properties["engine_installer.port"]; ok {
 			info = fmt.Sprintf("%sAccess the Demo VCH Installer at:\nhttps://%s:%s\n", info, ip.String(), port)
 		}
-		if port, ok := ovf.Properties["cluster_manager.port"]; ok {
-			info = fmt.Sprintf("%sAccess the Kubernetes on vSphere daemon at:\nhttps://%s:%s\n", info, ip.String(), port)
-		}
 	}
 	info = fmt.Sprintf("%s\nAccess the VIC Product Documentation at:\nhttps://vmware.github.io/vic-product/#documentation\n", info)
 	info = fmt.Sprintf("%s\n\nPress the right arrow key to view network status...", info)
```
Comment (maplain) [inline installer/ovatools/vic-ova-ui/main.go]: https://github.com/vmware/vic-product/pull/369/files#diff-055c4fed85f7c36e5a06501ffb476452L113

Comment (andrewtchin) [inline installer/ovatools/vic-ova-ui/main.go]: Do I need to change the network status line or something?

Comment (maplain) [inline installer/ovatools/vic-ova-ui/main.go]: oh, I mean I modified the display format back then.  You may want to replace '\n\nPress the right key ...' to '\n\n\n\n\t\t\t\t\t\tPress the right key'.
check out @morris-jason 's comment [here](https://github.com/vmware/vic-product/pull/369)

**Pull request merged**